harness/determined/common/experimental/determined.py

```diff
@@ -26,20 +62,62 @@ def __init__(
         user: Optional[str] = None,
     ):
         self._session = Session(master, user)
+        self._auth = auth.Authentication.instance()
```

Comment:
It looks like you do not need to save this object to `self`, since it doesn't look like you are using it anywhere outside of `__init__()`

Reply:
removed from self as its only used for initializing Swagger-api classes